openmdao/core/explicitcomponent.py

```diff
@@ -344,7 +344,7 @@ def _linearize(self, jac=None, sub_do_ln=False):
         Parameters
         ----------
         jac : Jacobian or None
-            If None, use local jacobian, else use assembled jacobian jac.
+            Ignored.
```

Comment:
im guessing the argument is needed for implicit comps then? otherwise why not just remove it?

Reply:
It is used for implicit comps, but even if it weren't we'd still need it because it's needed by groups.